libs/sdk-ui-dashboard/src/presentation/widget/insight/DefaultDashboardInsight/DefaultDashboardInsight.tsx

```diff
@@ -65,6 +68,15 @@ export const DefaultDashboardInsight = (props: IDashboardInsightProps): JSX.Elem
     const [isVisualizationLoading, setIsVisualizationLoading] = useState(false);
     const [visualizationError, setVisualizationError] = useState<GoodDataSdkError | undefined>();
 
+    const handlePushData = useCallback(
+        (data: IPushData): void => {
+            if (onAvailableDrillTargetsReceived && data?.availableDrillTargets) {
+                onAvailableDrillTargetsReceived(data?.availableDrillTargets);
```

Comment:
Nitpick: Optional chaining should not be necessary inside the condition

Reply:
Done